data/base/base-route.php

```diff
@@ -0,0 +1,258 @@
+<?php
+namespace Elementor\Data\Base;
+
+use WP_REST_Server;
+
```

Comment:
Explain the class purpose.

Reply:
Handled: https://github.com/elementor/elementor/pull/12820/commits/8f488da8e79062fddfc8efbab4b94872b5d4f3e0